editor/src/components/canvas/canvas-strategies/strategies/reparent-helpers/reparent-property-changes.ts

```diff
@@ -183,6 +180,7 @@ export function getAbsoluteReparentPropertyChanges(
 
   return [
     adjustCssLengthProperties('always', target, newParentFlexDirection, edgePropertiesToAdjust),
+    setProperty('always', target, PP.create('style', 'position'), 'absolute'),
```

Comment:
This really shouldn't be needed here. We have a strategy for converting to absolute before reparenting, which is why this hasn't been needed before.

Reply:
removed in https://github.com/concrete-utopia/utopia/pull/4344/commits/8699f0e10b0af192fd8f3bf4208eadc7d61f3d1f